metabase/metabase
# Usage insight links on questions and dashboards

Closes https://github.com/metabase/metabase/issues/38474 

### Description
This PR adds the `Usage Insights` option to the menus in Questions and Dashboards that will take you to the appropriate overview dashboard inside Instance Analytics for the given entity. Links will only appear in EE, and for users that have access to the Instance Analytics collection. 

### How to verify

1. As an Admin (Or a user with access to IA), go to any question
2. In the query builder header, click `...` and select "Usage Analytics". You should be brought to the Questions Overview dashboard filtered on that specific question you were just on
3. Try the same thing on Dashboards

### Demo
![chrome_stTzOLeDgz](https://github.com/metabase/metabase/assets/1328979/aa84f357-6533-4172-a04f-77393ddbf80f)

### Checklist
